pinc/filter_project_list.inc

```diff
@@ -151,23 +151,27 @@ class DifficultyElement extends ProjectFilterElement
 {
     public function echo_active_html_control()
     {
-        echo "<p><b>$this->label:</b> ";
-        echo "<label for='diff-all'>" . pgettext("all difficulties", "All") . "</label><input type='checkbox' name='difficulty[]' id='diff-all' value=''";
+        echo "<div class='flex_row'>";
+        echo "<div class='entry'><b>$this->label:</b></div>";
```

Comment:
We may want to html encode this to cover any cases where it needs encoding now or in the future or in a different language

Reply:
Good eye - I fixed this, the one below, and a few other existing instances in the file.